community/index/src/main/java/org/neo4j/index/internal/gbptree/ConsistencyChecker.java

```diff
@@ -17,22 +17,23 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.index.gbptree;
```

Comment:
@tinwelint @burqen i'm surprised a bit to see that all classes are moved to internal package now; was excepting to see there only things that should be internal to this module and in public package something that should be visible and used outside of this module